src/utils/format.ts

```diff
@@ -94,7 +95,7 @@ export const formatPSCForDisplay = (psc: PersonOfSignificantControl): PersonOfSi
 export const formatSecretaryList = (officers: ActiveOfficerDetails[]): any[] => {
   const secretaryList = new Array(0);
   for (const officer of officers) {
-    if (officer.role === "SECRETARY" && officer.isCorporate === false ) {
+    if (officer.role === OFFICER_ROLE.SECRETARY && officer.isCorporate === false ) {
```

Comment:
can you convert officer.role to capitals and keep the enum as capital, we won't have to worry about matching case then